### ISSUE TITLE
[Translation] Strain On Reality

### ISSUE THREAD

**Issue by andrex47** [NPBruce/valkyrie#1462]
[Localization.Portuguese.txt](https://github.com/NPBruce/valkyrie/files/6083495/Localization.Portuguese.txt)

### What kind of translation do you want to add (mark with an X, e.g. [X] Descent)?
- [X] Mansions of Madness
- [ ] Descent

### Strain On Reality

We also have a sheet with the translation of the images for this scenario which can be acessed here:
https://drive.google.com/file/d/1ASVu4yHfqRGQX97sbcHVNg03z-SaMll5/view?usp=sharing
We also have audio files:
https://drive.google.com/drive/folders/1NQ7Y_5ZJuE_k0msb16-g03YWuDW6ZZOP?usp=sharing

**Comment by vidimus** [comment]
@scrubbless

**Comment by scrubbless** [comment]
Awesome, thank you. I am looking at this now

**Comment by scrubbless** [comment]
Can you let me know who you would like to add to the credits? Voice actor/translators. I'll add it to the "Authors" section of the scenario in each language.

**Comment by scrubbless** [comment]
Just adding this updated localisation file so I don't lose sight of it

[Localization.Portuguese.txt](https://github.com/NPBruce/valkyrie/files/6085939/Localization.Portuguese.txt)

**Comment by scrubbless** [comment]
My apologies for the delay in this, I have been working on the images for this scenario. I am about 70% complete but have not had much spare time! I'll get this up as soon as I can.

**Comment by Bigdadd1** [comment]
> My apologies for the delay in this, I have been working on the images for this scenario. I am about 70% complete but have not had much spare time! I'll get this up as soon as I can.

Hello scrubbles I would like to know how the images update for the strain on reality is going, do you have any idea when the scenario will be in portuguese??

**Comment by scrubbless** [comment]
My apologies, it's been insane the past months. I promise I will get to this.

**Comment by Bigdadd1** [comment]
no problems, buddy. I hope everything is calmer now. When you're done,
just let me know.
Hugs

[image: image.png]


Em ter., 28 de set. de 2021 às 19:26, scrubbless ***@***.***>
escreveu:

**Comment by scrubbless** [comment]
I'm so sorry this has taken so long, i've now done all of the images.

What names would you like me to put for credit in the scenario description?

**Comment by Bigdadd1** [comment]
TY for your effort!! The name It can be: MoM group.

Em sex, 5 de nov de 2021 12:08, scrubbless ***@***.***>
escreveu:

**Comment by scrubbless** [comment]
Again my sincerest apologies that it has taken me this look to update the scenario to include your groups translation, especially considering how much of a monumental undertaking this scenario must have been to translate.

Please pass on my gratitude to your group for taking the time translating this and being patient with me.

**Comment by scrubbless** [comment]
I'll update the BGG post to announce the translation then start on the Spanish version that @vidimus has worked on
😁 

Closing this issue!

**Comment by Bigdadd1** [comment]
no need to apologize for anything friend. Your work is excellent and
we are going to enjoy the scenery a lot.
A big hug!!!


Em sex., 5 de nov. de 2021 às 12:39, scrubbless ***@***.***>
escreveu:

**Comment by scrubbless** [comment]
Hi @Bigdadd1 / @andrex47 

I've realised that I have not uploaded the correct audio files to the scenario, looking at the google link there are a lot of audio files in there. 

I only have audio for the 3 starting intros (part1.ogg / part 2.ogg / part3.ogg) at present, so could you identify which ones these are so I can update the scenario. I have a feeling that it is one audio file - IntroduçãoStrain.ogg. If so could you split it into three parts (there are three pages of text in the intro and each should be their own file so that players can skip the audio if they wish and keep sync - also there are tile placements between the second part and the 3rd part).

If they are all audio or this scenario I would be happy to add them, however I would also need these split as audio files for each page of text in the splash screen. It will also take a bit longer to add them as i'd need to do some overhauls to the scenario to incorporate them (as I'd need to add placeholder sounds files to all other translations and the English verison).

**Comment by vidimus** [comment]
Hello, @scrubbless 
Is there any chance to add voiceover to the endings? I'm trying to get a pair of semi-professional actors to lend their voices for this scenario (I really think this scenario deserves it) and it's a pity that only intro has voice but not the different endings.

**Comment by scrubbless** [comment]
> Hello, @scrubbless Is there any chance to add voiceover to the endings? I'm trying to get a pair of semi-professional actors to lend their voices for this scenario (I really think this scenario deserves it) and it's a pity that only intro has voice but not the different endings.

Yes I can. Ertainly do this, I can add some placeholder music to replace the voice overa that I don't have translated. I will talk to Steve to see if he can voice the outros too. This will just take time.

My only request if you get this done before I get to updating the scenario - can you make the audio files separate files for each 'page'. Some endings display one page, which is generic for multiple endings then a second page for the specific ending.

I would slice audio files up myself but sadly I wouldn't understand the readings enough to know when to split them 😄

**Comment by scrubbless** [comment]
> Hi @Bigdadd1 / @andrex47
> 
> I've realised that I have not uploaded the correct audio files to the scenario, looking at the google link there are a lot of audio files in there.
> 
> I only have audio for the 3 starting intros (part1.ogg / part 2.ogg / part3.ogg) at present, so could you identify which ones these are so I can update the scenario. I have a feeling that it is one audio file - IntroduçãoStrain.ogg. If so could you split it into three parts (there are three pages of text in the intro and each should be their own file so that players can skip the audio if they wish and keep sync - also there are tile placements between the second part and the 3rd part).
> 
> If they are all audio or this scenario I would be happy to add them, however I would also need these split as audio files for each page of text in the splash screen. It will also take a bit longer to add them as i'd need to do some overhauls to the scenario to incorporate them (as I'd need to add placeholder sounds files to all other translations and the English verison).

Just want to keep this request on the forfront of the chat!

**Comment by Bigdadd1** [comment]
I texted the fellow who made the audios and he'll try to organize them
to send to you.
Thanks you!



Em seg., 8 de nov. de 2021 às 08:40, scrubbless ***@***.***>
escreveu:

**Comment by scrubbless** [comment]
Thank you. The existing files could just be split/cut I suspect. 

I just don't know the language to know when to cut it correctly

**Comment by vidimus** [comment]
> >
> Yes I can certainly do this, I can add some placeholder music to replace the voice over that I don't have translated. I will talk to Steve to see if he can voice the outros too. This will just take time.
> 
> My only request if you get this done before I get to updating the scenario - can you make the audio files separate files for each 'page'. Some endings display one page, which is generic for multiple endings then a second page for the specific ending.
> 
> I would slice audio files up myself but sadly I wouldn't understand the readings enough to know when to split them smile

Hello, @scrubbless 
Actually, I've already done it: the intro have been sent as 3 separate pages (3 audio files), and endings too (a big text plus a two sentences final text).

**Comment by scrubbless** [comment]
> I texted the fellow who made the audios and he'll try to organize them to send to you. Thanks you! Em seg., 8 de nov. de 2021 às 08:40, scrubbless ***@***.***> escreveu:


@andrex47 @Bigdadd1 

Any luck with splitting those Audio files?

**Comment by Bigdadd1** [comment]
I just sent a message to the creator of the audios, waiting for his response.

TY!


Em qua., 17 de nov. de 2021 às 07:31, scrubbless ***@***.***>
escreveu:

**Comment by Bigdadd1** [comment]
Scrubbles, I talked to the creator of the audios and he said that you
can only put in the audios of the scenario intro. Because he's having
trouble seeing it now


Em seg., 22 de nov. de 2021 às 19:40, Paulo Santos <
***@***.***> escreveu:

**Comment by scrubbless** [comment]
Hi @Bigdadd1 

Are the audio files on the Google drive link? I need the intro split into 3 files (part1.ogg, part2.ogg and part3.ogg).

The link above only shows one file that looks like the intro
![Screenshot_20211125-194729_Drive](https://user-images.githubusercontent.com/32058089/143494485-0b4f238f-0f5a-4a74-bd4f-68f24662734b.jpg)

**Comment by Bigdadd1** [comment]
I spoked with the creator of the audios he will try to do the separation


Em qui., 25 de nov. de 2021 às 16:49, scrubbless ***@***.***>
escreveu:
